mutable_tree.go

```diff
@@ -616,19 +616,31 @@ func (tree *MutableTree) LoadVersion(targetVersion int64) (int64, error) {
 // LoadVersionForOverwriting attempts to load a tree at a previously committed
 // version, or the latest version below it. Any versions greater than targetVersion will be deleted.
 func (tree *MutableTree) LoadVersionForOverwriting(targetVersion int64) (int64, error) {
-	latestVersion, err := tree.LoadVersion(targetVersion)
+	return tree.LoadVersionForOverwritingWithMode(targetVersion, false)
+}
+
+func (tree *MutableTree) LoadVersionForOverwritingWithMode(targetVersion int64, fastMode bool) (int64, error) {
+	var latestVersion int64
+	var err error
+	if !fastMode {
+		latestVersion, err = tree.LoadVersion(targetVersion)
+	} else {
+		latestVersion, err = tree.LazyLoadVersion(targetVersion)
```

Comment:
There is a param name clash:
* seams that we want to use `fastMode` when we don't want to use `fastCache`
* so let's rename `fastMode` parameter to something different, eg `lazy bool` or `noFastCache bool`.

Reply:
Actually this flag is mainly related with rollback, should we rename to `FastRollback ` or `OfflineRollback`

Reply:
Make sense, then `OfflineRollback` might better since [assumption](https://github.com/cosmos/iavl/blob/e0cfa4279fa59d18071a001b96bc2aea951638db/nodedb.go#L512) is based on offline and re-index on restart.